### PR TITLE
batch update recent spec changes

### PR DESCRIPTION
**Pull request by lisaqyli**
commits that goes into this task: https://github.com/fiatconnect/specification/commit/a8b8b638ac14579451c4cbccfff02029e046901a
https://github.com/fiatconnect/specification/commit/814e6c7214866d0b28ec00addcd2568800d7e5a0
https://github.com/fiatconnect/specification/commit/e3bcaaab5bcec5c13bfcbbe52e31969b387a36a2

also updated TransferStatusEnum

Fixes #2449

### CODE REVIEW

**Comment by jh2oman** [comment]
There were a few more added just this morning. Can you update to include those too? https://github.com/fiatconnect/specification/commits/main

File: src/index.ts [hunk 1]
```diff
@@ -327,6 +336,28 @@ export interface AccountNumber {
   country: string,
   fiatAccountType: FiatAccountType.BankAccount
 }
+export interface MobileMoney {
+  accountName: string;
+  institutionName: string;
```
[the remark this answers is not in the document]
Comment (lisaqyli) [reply src/index.ts]: idk if  both accountName and institutionName will be required for all future fiat schema. I don't really want to make the assumption for now.

[the remark this answers is not in the document]
Comment (cajubelt) [reply src/index.ts]: https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#832-fiat-account-schemas

File: src/index.ts [hunk 2]
```diff
@@ -331,28 +331,28 @@ export enum FiatAccountType {
   DuniaWallet = 'DuniaWallet'
 }
 
-export interface AccountNumber {
-  accountName: string,
-  institutionName: string,
+interface requiredFiatAccountSchemaFields {
```
Comment (cajubelt) [inline src/index.ts:334]: please use pascal case here

Comment (cajubelt) [reply src/index.ts:334]: (first letter capitalized)

**Review by cajubelt — APPROVED** [review]
approved assuming the casing is fixed. thanks

**Pull request merged**